release.nix

```diff
@@ -76,6 +77,63 @@ let
     connectScripts.testnet.explorer = [ "x86_64-linux" "x86_64-darwin" ];
   } skipPackages;
   mapped = mapTestOn platforms;
+
+  nix-tools-toolchain = {
+    nix-tools.libs = removeAttrs {
+      # nix-tools toolchain: Libraries
+      cardano-sl            = supportedSystems;
+      cardano-sl-auxx       = supportedSystems;
+      cardano-sl-chain      = supportedSystems;
+      cardano-sl-core       = supportedSystems;
+      cardano-sl-crypto     = supportedSystems;
+      cardano-sl-db         = supportedSystems;
+      cardano-sl-generator  = supportedSystems;
+      cardano-sl-infra      = supportedSystems;
+      cardano-sl-networking = supportedSystems;
+      cardano-sl-tools      = supportedSystems;
+      cardano-sl-util       = supportedSystems;
+      cardano-wallet        = supportedSystems;
+      cardano-sl-x509       = supportedSystems;
+      turtle                = supportedSystems;
+    } skipPackages;
+    nix-tools.exes = removeAttrs {
+      # nix-tools toolchain: Executables
+      # these will usually implicitly build their
+      # library as they depend on it.
+      cardano-sl-tools             = supportedSystems;
+      cardano-sl-generator         = supportedSystems;
+      cardano-sl-tools-post-mortem = supportedSystems;
+      cardano-wallet               = supportedSystems;
+    } skipPackages;
+    # nix-tools toolchain: Tests
+    nix-tools.tests =
+      removeAttrs
+        (lib.mapAttrs (_: lib.mapAttrs (_: _: supportedSystems))
+          (lib.filterAttrs (n: v: builtins.match ".*cardano-sl.*" n != null && v != null)
```

Comment:
Is this more or less the same as `localLib.isCardanoSL`, or must the regexp be different?